sp-files/.sphinx/.markdownlint.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I've had more of a look and I know this list comes from @evilnick but I think we need to discuss some of these items before implementing things for the whole company without discussion.

There are also some rules missing that I think could be beneficial.

Reply:
As Maksim said, we can continue to iterate.